providers/route53/route53Provider.go

```diff
@@ -305,3 +385,31 @@ func (r *route53Provider) EnsureDomainExists(domain string) error {
 	return err
 
 }
+
+func (r *route53Provider) waitUntilNameserversUpdate(operationId *string) error {
```

Comment:
I'm not exactly sure what the possibilities are for operations. [According to the API](https://github.com/aws/aws-sdk-go/blob/6a6d169fb0936d588240e44238579aaa32842863/models/apis/route53domains/2014-05-15/api-2.json#L1029), operations can still still fail. I'm not sure what might cause failures, though.

Validation failures (like having duplicate nameservers) will fail before even generating an operation id, so I don't know what scenarios there are for failure after the operation has been created.